apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-logs-sidebar.tsx

```diff
@@ -58,26 +67,19 @@ export const TestWorkflowLogsSidebar = ({ transactionId, workflow }: TestWorkflo
 
   return (
     <aside className="flex h-full max-h-full flex-1 flex-col overflow-auto">
-      {transactionId && !activityId ? (
-        <div className="flex h-full items-center justify-center">
-          <div className="flex flex-col items-center gap-4">
-            <Loader2 className="size-8 animate-spin text-neutral-500" />
-            <div className="text-foreground-600 text-sm">Waiting for activity...</div>
-          </div>
-        </div>
-      ) : activityId ? (
+      {transactionId ? (
         <>
           <ActivityPanel>
             {isPending ? (
               <ActivitySkeleton />
             ) : error || !activity ? (
               <ActivityError />
             ) : (
-              <>
+              <React.Fragment key={activityId}>
```

Comment:
Play the fade-in animation when the `activityId` changes, for example, when viewing parent details of the merged activity. So like when the view is the same but the details are different.